src/gdcdictionary/schemas/demographic.yaml

```diff
@@ -1180,16 +1180,16 @@ properties:
     $ref:
       - "_terms.yaml#/race/common"
     enum:
-      - white
       - american indian or alaska native
-      - black or african american
       - asian
+      - black or african american
       - native hawaiian or other pacific islander
       - other
+      - unknown
+      - white
```

Comment:
white should bubble up to group the 'unknown' and 'Unknown' together. Add 'not allowed to collect' to the documentation regarding ordering.

Reply:
The "unknown" value has been deprecated, meaning that we are in the process of migrating it to "Unknown" and it should go away in the next release.  Good idea with adding "not allowed to collect" to the document.

Reply:
Resolved ordering of enums, added 'not allowed to collect' sort guideline to the documentation.